### ISSUE TITLE
Ambiguous ordering of joint data

### ISSUE THREAD

**Issue by DanielChaseButterfield** [UMich-CURLY/deep-contact-estimator#3]
I've been reading your paper, "Legged Robot State Estimation using Invariant Kalman Filtering and Learned Contact Events", as well as investigating the code found here, in order to determine the order of the joint data found in your provided Contact dataset.

Unfortunately, so far I've been unsuccessful. We would like to use this dataset in our own research, but we need to know which joint index corresponds to which joint on the robot. Would you be able to clearly define the order of the joint data?

![image](https://github.com/UMich-CURLY/deep-contact-estimator/assets/86738206/7f4abc5c-4d5d-4fee-9e4e-9689348e670d)

**Comment by DanielChaseButterfield** [comment]
After talking with @tzuyuan, the ordering follows what is outlined here: https://github.com/mit-biomimetics/Cheetah-Software/blob/master/documentation/getting_started.md

Basically, the first 3 are front right, the next are front left, the next are back right, and the last three are back left.

In terms of joint order, it goes from the abductor/adductor joint to the hip joint to the knee joint.
Or, if you are using hip, thigh, and knee terminology, it goes from the hip joint to the thigh joint to the knee joint.
Basically, the joint order starts from the robot base and travels down the legs to the foot.
